### DISCUSSION TITLE
Old ARM build... trying to get running again after updating from Ubunut 16 LTS to 22LTS

### DISCUSSION THREAD

**Discussion by coacharnold1** [automatic-ripping-machine/automatic-ripping-machine#1100] [General]
HI folks ....   
     I recently found my ARM build  from a few years ago. I had pulled the hard drive to use the machine as something else.   In the years since I have attempted to run ARM on a few other  builds but I always get stuck on the UDEV rules to start the thing going on DVD/CD insert.    SO I put the the old drive back in it's machine, updated to Ubuntu 18 > 20 > 22 LTS so that I could keep everthing sort of the same.     I completed the upgrade, rebooted and dropped a DVD in the machine ,,, and it fired up .   It sounded like it was going to rip the DVD,  but Handbreak exited after about 5 min ...    I dropped a CD in and it began a perfect rip to flac.. It didn't pick up any ifo from CDDB ....  which has me concerned that there something missing ...  so 2 questions ...

1:  DO I need to update Handbrake,  it's on 1.51   but is there a license I'm missing or something?

2:  Is there something I need to update to get CDDB to get cd info 

Mind you all,  Ive done nothing to even try to upgrade anything to do with ARM,  I was hoping to get it working as is.   WOuld I be better installing the docker image?  Is that easyier to get working with UDEV to auto run when a DVD is inserted?

THanks ...  sorry for the long post 

Tim

**Comment by shitwolfymakes** [comment]
We no longer support thick client installations like this. Please delete the installation and install via [Docker](https://github.com/automatic-ripping-machine/automatic-ripping-machine/wiki/docker)
